sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -5310,6 +5321,9 @@ class SQLConf extends Serializable with Logging with SqlApiConf {
     getConf(SQLConf.LEGACY_NEGATIVE_INDEX_IN_ARRAY_INSERT)
   }
 
+  def legacyRaiseErrorWithoutErrorClass: Boolean =
```

Comment:
where do we read this conf?

Reply:
Ah, I got interrupted and lost in the regression.